shamir/shamir.go

```diff
@@ -88,57 +88,32 @@ func div(a, b uint8) uint8 {
 		panic("divide by zero")
```

Comment:
Should this if statement use subtle.ConstantTimeByteEq?

Reply:
No, that case should never get hit, so having it be a different timing in the panic case from the normal case is fine.  I'll add a comment about it though.